### PR TITLE
#367-added close button on new posts

### LINKED ISSUE #367: Close button missing on create new post
### Description

After clicking create New post if we want to back there is no close button 
Pls assign this to me under gssoc23

### Steps to reproduce

Create new post 

### Screenshots

Screenshot_2023-05-27-16-07-21-61_40deb401b9ffe8e1df2f1cc5ba480b12

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Soulpurr**
fixes #367 
So i have added a cross on create new posts which closes the current window
[scrnli_27_05_2023_23-53-33.webm](https://github.com/narayan954/dummygram/assets/92094765/999b332c-447a-4e6f-b849-e75f144661e0)

### CODE REVIEW

**Comment by Soulpurr** [comment]
@narayansoni pls approve this pr

File: src/components/ImgUpload.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import "./imgPreview.css";
 import { LinearProgress, TextField } from "@mui/material";
 import React, { useRef, useState } from "react";
 import { db, handleMultiUpload, storage } from "../lib/firebase";
-
+import {AiOutlineClose} from 'react-icons/ai'
```
Comment (narayan954) [inline src/components/ImgUpload.jsx:6]: extra import

File: src/components/ImgUpload.jsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ function ImgUpload(props) {
   const imgInput = useRef(null);
   const [imagePreviews, setImagePreviews] = useState([]);
   const [isValidimage, setisValidimage] = useState(true);
+  const [close, setclose] = useState(false)
```
Comment (narayan954) [inline src/components/ImgUpload.jsx:19]: unnecessary state

**Review by narayan954 — CHANGES_REQUESTED** [review]
minor changes

File: src/components/ImgUpload.jsx (hunk 2 above)
Comment (Soulpurr) [reply src/components/ImgUpload.jsx:19]: Ohh just missed will fix it

**Comment by Soulpurr** [comment]
@narayan954  can you pls verify that the changes are reflected or not

**Comment by narayan954** [comment]
> @narayan954 can you pls verify that the changes are reflected or not

yes all right!

**Comment by Soulpurr** [comment]
Pls approve it then🥹

**Comment by narayan954** [comment]
> Pls approve it then🥹

there you go!

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution!

**Pull request merged**